extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -1,20 +1,26 @@
 /**
- * Copyright (c) 2014-2017 by the respective copyright holders.
+ * Copyright (c) 2017 by the respective copyright holders.
```

Comment:
Why have you removed the `2014-`?
Isn't the range added by the licence maven plugin that we use to update the header and to ensure every file is using the correct one?